### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by stweil** [comment]
I'm not sure whether the fixes in `tests` are fine – that's why I separated them in an extra commit.

File: tests/unit/mock_lslpp_Lc [hunk 1]
```diff
@@ -85,7 +85,7 @@ X11.msg.en_US:X11.msg.en_US.vsm.rte:7.1.3.0: : :C: :Visual Sys Mgmt. Helps & Msg
 X11.samples:X11.samples.apps.clients:7.1.3.30: : :C: :AIXwindows Sample X Consortium Clients Binary/Source : : : : : : :1:0:/:1441
 X11.samples:X11.samples.common:7.1.0.0: : :C: :AIXwindows Imakefile Structure for Samples : : : : : : :1:0:/:1115
 X11.samples:X11.samples.lib.Core:7.1.3.30: : :C: :AIXwindows Sample X Consortium Core Libraries Binary/Source : : : : : : :1:0:/:1441
-X11.vsm:X11.vsm.lib:7.1.0.0: : :C: :Visual System Managment Library: : : : : : :1:0:/:1036
```
Comment (nickanderson) [inline tests/unit/mock_lslpp_Lc]: This is from output on an AIX system, so I think that typo should stay.

File: tests/unit/mock_nimclient_showres [hunk 2]
```diff
@@ -197,7 +197,7 @@ X11.samples:X11.samples.fnt.util:7.1.3.30::I:T:::::N:AIXwindows Sample Font Serv
 X11.samples:X11.samples.lib.Core:7.1.3.30::I:C:::::N:AIXwindows Sample X Consortium Core Libraries Binary/Source ::::0:1441:
 X11.samples:X11.samples.rgb:7.1.0.0::I:T:::::N:AIXwindows Sample Color Database Source ::::0:1115:
 X11.vfb:X11.vfb:7.1.3.0::I:T:::::N:Virtual Frame Buffer Software for AIXwindows ::::0:1341:
-X11.vsm:X11.vsm.lib:7.1.0.0::I:C:::::N:Visual System Managment Library::::0:1036:
```
Comment (nickanderson) [inline tests/unit/mock_nimclient_showres]: Also from system output, I think it should remain the same

File: tests/unit/mock_nimclient_showres [hunk 3]
```diff
@@ -286,7 +286,7 @@ bos.clvm:bos.clvm.enh:7.1.3.30::I:C:::::b:Enhanced Concurrent Logical Volume Man
 bos.compat:bos.compat.cmds:7.1.3.0::I:T:::::N:AIX 3.2 Compatibility Commands ::::0:1341:
 bos.compat:bos.compat.libs:7.1.0.0::I:T:::::N:AIX 3.2 Compatibility Libraries ::::0:1115:
 bos.compat:bos.compat.links:7.1.3.0::I:T:::::N:AIX 3.2 to 4 Compatibility Links ::::0:1341:
-bos.compat:bos.compat.net:7.1.0.0::I:T:::::N:AIX 3.2 TCP/IP Compatability Commands ::::0:1115:
```
Comment (nickanderson) [inline tests/unit/mock_nimclient_showres]: This one too.

**Comment by nickanderson** [comment]
This looks good to me except for the changes to the aix mock tests. Can you revert those?

**Comment by stweil** [comment]
Updated PR, now without the 2nd commit (which included the changes in `test`).

**Comment by nickanderson** [comment]
Thank you!

**Pull request merged**